Libraries/Text/Text/RCTTextView.m

```diff
@@ -104,7 +104,12 @@ - (NSString *)description
   NSString *superDescription = super.description;
```

Comment:
What is the `superDescription` that was assumed to have a semi colon at the end?

Reply:
I don't think it's necessarily at the end, just that iOS provides a fair amount more information in its description.

Reply:
I'm still a little confused why we would try and replace a semicolon in the first place instead of just appending it at the end.